src/main/java/com/users/endpoint/UserEndPoint.java

```diff
@@ -0,0 +1,40 @@
+package com.users.endpoint;
+
+import com.users.dto.UserDto;
+import com.users.entities.User;
+import com.users.service.UserService;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+@AllArgsConstructor
+@RestController
+@RequestMapping("/user")
+@Slf4j
+public class UserEndPoint {
```

Comment:
de obicei controllerele se denumesc UserController sau UserResource

Reply:
am vazut ca astia asa foloseau la proiecte si am zis ca poate asa trebuie ca stiu ca noi le am facut cu controller. Am corectat acum

Reply:
probabil e in functie de proiect, da eu am vazut doar cu variantele astea 2